src/utils/string/StringUtils.js

```diff
@@ -247,7 +247,7 @@ const helperGetMemberName = (member) => {
   const isAnonymous = Boolean(member?.anon_user_info_emoji_name);
 
   return isAnonymous
-    ? `Anonymous ${member?.anon_user_info_emoji_name}`
+    ? `${member?.anon_user_info_color_name} ${member?.anon_user_info_emoji_name}`
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a hardcoded "Anonymous" to `member?.anon_user_info_color_name` is a good one, as it allows for more flexibility in the naming of anonymous users. However, there should be a fallback value in case `member?.anon_user_info_color_name` is undefined or null.
```diff
-    ? `${member?.anon_user_info_color_name} ${member?.anon_user_info_emoji_name}`
+    ? `${member?.anon_user_info_color_name ?? 'Anonymous'} ${member?.anon_user_info_emoji_name}`
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->